lib/nimble_template/exception/mock_hex_package_required.ex

```diff
@@ -0,0 +1,3 @@
+defmodule NimbleTemplate.Exception.MockHexPackageRequiredException do
```

Comment:
Should we place this exception under `NimbleTemplate.Hex` namespace?

```suggestion
defmodule NimbleTemplate.Hex.Exception.MockHexPackageRequiredException do
```

Reply:
Based on our structure on the compass, we prefer to keep the exception outside the context @hanam1ni 

<img width="902" alt="image" src="https://user-images.githubusercontent.com/11751745/203099320-44bc1d35-4e1c-40c4-9bb3-7ac8c15011a3.png">